### PR TITLE
AKepska #341 [5] Update app year 2023

### LINKED ISSUE #341: [5] Update app year 2023

### PR DESCRIPTION
**Pull request by AKepska**
Udało mi się zmodyfikować też ekran powitalny.

### CODE REVIEW

File: elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java [hunk 1]
```diff
@@ -472,7 +472,7 @@ public static BlockReason getByNumber(String blockReason) {
 	public static final Boolean DATA_BASE_REGISTER_DEFAULT_FLAG = false;
 	public static final Integer DATA_BASE_ADMIN_ID = 0;
 	public static final String DATA_BASE_ADMIN_LOGIN = "admin";
-	public static final String DATA_BASE_ADMIN_PASSWORD = "f7e3c24e1a04758097f69be41aa3cf18";
+	public static final String DATA_BASE_ADMIN_PASSWORD = "b86101561d1558c5b2015174070e26b8";
```
Comment (JaRutkowski) [inline elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java]: Please revert as it's the change only for the local development env. 🗑️

File: elibrary-hibernate/pom.xml [hunk 2]
```diff
@@ -40,12 +40,12 @@
 					<propertyFile>liquibase.properties</propertyFile>
 					<changeLogFile>src/main/resources/db-changelog-master.xml</changeLogFile>
 					<!-- comment if running locally -->
-					<url>
-						jdbc:postgresql://ec2-46-137-121-216.eu-west-1.compute.amazonaws.com/dfsnu6tveajrq7?sslmode=require
-					</url>
-					<username>ppqrmsqersyeto</username>
-					<password>39f03b52de7505db0046939040eb112254bd4560779165c62a2792b9bfe65cfb</password>
-					<promptOnNonLocalDatabase>false</promptOnNonLocalDatabase>
+<!--					<url>-->
```
Comment (JaRutkowski) [inline elibrary-hibernate/pom.xml]: Please revert as it's the change only for the local development env. 🗑️

File: elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml [hunk 3]
```diff
@@ -1,5 +1,14 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="Mój (generated)" id="1593444482106-0">
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml]: Please revert as it's the change only for the local development env. 🗑️

File: elibrary-hibernate/src/main/resources/db/changelog/truncate-all-cascade-changelog.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
 	<changeSet author="Jakub.Rutkowski" id="0000000019">
 		<comment>Clear any existing legacy data in the tables</comment>
 		<sql splitStatements="true">
-			TRUNCATE TABLE public.com_authorization CONTINUE IDENTITY CASCADE;
+			-- TRUNCATE TABLE public.com_authorization CONTINUE IDENTITY CASCADE;
```
Comment (JaRutkowski) [inline elibrary-hibernate/src/main/resources/db/changelog/truncate-all-cascade-changelog.xml]: Please revert as it's the change only for the local development env. 🗑️

**Review by JaRutkowski — CHANGES_REQUESTED** [review]
PSB - comments 💬

File: elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java (hunk 1 above)
Comment (AKepska) [reply elibrary-core/source/src/controller/utils/com/javafee/elibrary/core/common/Constants.java]: Fixed

File: elibrary-hibernate/src/main/resources/db/changelog/truncate-all-cascade-changelog.xml (hunk 4 above)
Comment (AKepska) [reply elibrary-hibernate/src/main/resources/db/changelog/truncate-all-cascade-changelog.xml]: Fixed 🔨

File: elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml (hunk 3 above)
Comment (AKepska) [reply elibrary-hibernate/src/main/resources/db/changelog/db-changelog-20210816101031UTC.xml]: Fixed 🔨

File: elibrary-hibernate/pom.xml (hunk 2 above)
Comment (AKepska) [reply elibrary-hibernate/pom.xml]: Fixed 🔨

**Review by JaRutkowski — APPROVED** [review]
🚀

**Pull request merged**